virttest/virt_vm.py

```diff
@@ -1287,13 +1287,19 @@ def get_memory_size(self, cmd=None, timeout=60, re_str=None):
             mem_size = 0
             for m in mem:
                 mem_size += int(m)
-            if "GB" in mem_str:
```

Comment:
@xutian mem_str is return by "dmidecode -t 17 | awk -F: '/Size/ {print $2}'", not for "cat /proc/meminfo", so mem_str like this:
"
 4096 MB
 4096 MB
"
so, cannot use mem_str,  need to catch the unit.